vendor.mod

```diff
@@ -80,9 +80,9 @@ require (
 	github.com/moby/sys/userns v0.1.0
 	github.com/moby/term v0.5.2
 	github.com/morikuni/aec v1.0.0
+	github.com/opencontainers/cgroups v0.0.1
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0
-	github.com/opencontainers/runc v1.2.5
```

Comment:
This reminds me of an old issue of mine, which can probably be closed already, but definitely after this PR? 🤔 

https://github.com/moby/moby/issues/39166

Reply:
Not directly related; that ticket is more related to packaging (and http://github.com/docker/containerd-packaging). This is only a build-dependency on some of the Go libraries provided by runc.